modules/nf-core/bedtools/shift/meta.yml

```diff
@@ -0,0 +1,37 @@
+name: bedtools_shift
+description: Shifts each feature by specific number of bases
+keywords:
+  - bed
+  - shiftBed
```

Comment:
Can you add some more keywords?

Reply:
Like what?

Reply:
It's mainly because the linter will complain for modules that have so few keywords. A couple suggestions are: regions, fai, genome, base pairs...